src/accessors.d

```diff
@@ -389,6 +444,17 @@ private template generateAttributeString(uint attributes)
     }
 }
 
+private enum mayGainfullyStripConst(T) = !is(T == Unqual!T) && isAssignable!(Unqual!T, T);
```

Comment:
omit "Gainfully": the check only ensures that `const` can be stripped, but not whether this is gainful

Reply:
"gainfully" is checked via !is(T == Unqual!T), ie. whether T actually has any attributes to strip. If not, we can omit the stripping code for a cleaner accessor.